src/main/java/it/pn/frontend/e2e/model/Disservice.java

```diff
@@ -0,0 +1,19 @@
+package it.pn.frontend.e2e.model;
+
+import lombok.Builder;
+import lombok.Setter;
+
+@Builder
+@Setter
+public class Disservice {
+    private String dataDa;
+    private String dataA;
+
+    public String getDataA() {
+        return dataA.replace("ore", "").trim();
+    }
+
+    public String getDataDa() {
+        return dataDa.replace("ore", "").trim();
+    }
+}
```

Comment:
i don't see the point to have a specific Data Class for this purpose, just do the `replace()`  method in the related class.

Reply:
the disservice class currently contains dates, but in the future it could be used for other data

Reply:
done